File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& arr) {
        int n=arr.size();
        int count=0;
        int maxOnes=0;
        for(int i=0;i<n;i++){
            if(arr[i]==0){
                
                count=0;
            }
            else{
                count++;
            }
            maxOnes=max(maxOnes,count);
            
        }
     return maxOnes;  
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int mcnt=0;
        int cnt=0;
        int n=nums.length;
        for(int i=0;i<n;i++){
            if(nums[i]==1){
                cnt++;
                mcnt=Math.max(cnt,mcnt);
            }else{
                cnt=0;
            }



        }
        return mcnt;
        
    }
}